cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,28 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+    cy.visit('https://react-redux.realworld.io');
+  });
+
+  it('should click on Sign In link', () => {
```

Comment:
its simple test, u can use one "it" to cover all flow, make some changes in your code  to avoid extra tests

Reply:
thanks, already fixed it